cpp/llm_chat.cc

```diff
@@ -787,6 +900,33 @@ class LLMChat {
     return fsample_topp_from_prob_(logits_on_cpu_, top_p_, GetRandomNumber());
   }
 
+  /* Create custom string.split() function. */
+  std::vector<std::string> customSplit(std::string str, std::string separator) {
```

Comment:
CamelCase

Reply:
would you like to clarify on what CamelCase means in this scenario? The function name seems to follow the CamelCase

Reply:
Should be CustomSplit per Google C style

Reply:
changed